modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/user/mgt/UserImportLoggingTestCase.java

```diff
@@ -0,0 +1,244 @@
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.USER_STORE_DOMAIN;
+
+/**
+ * Test cases for the bulkUserImport log appender.
+ * This will check the logs are directed to the new file.
+ */
+public class UserImportLoggingTestCase extends ISIntegrationTest {
+
+    private static final Log log = LogFactory.getLog(UserImportLoggingTestCase.class);
+    private UserManagementClient userMgtClient;
+    private boolean isFilePresent = false;
+    private final String BULK_USER_FILE = getISResourceLocation() + File.separator + "userMgt"
+            + File.separator;
+    private final String LOG_FILE_LOCATION = Utils.getResidentCarbonHome() + File.separatorChar + "repository" +
+            File.separatorChar + "logs";
+
+    @BeforeClass
+    public void init() throws Exception {
+
+        super.init(TestUserMode.SUPER_TENANT_ADMIN);
+        userMgtClient = new UserManagementClient(backendURL, getSessionCookie());
+        PropertyConfigurator.configure(createLog4jConfiguration());
+    }
+
+    @Test(description = "Perform bulk user import and check the bulkuserimport log file is created.")
+    @SetEnvironment(executionEnvironments = {ExecutionEnvironment.STANDALONE})
+    public void testBulkUserImportLogFileIsPresent() throws RemoteException, UserAdminUserAdminException {
+
+        File bulkUserFile = new File(BULK_USER_FILE + BULK_USER_IMPORT_SUCCESS_FILE);
+
+        DataHandler handler = new DataHandler(new FileDataSource(bulkUserFile));
+        userMgtClient.bulkImportUsers(USER_STORE_DOMAIN, BULK_USER_IMPORT_SUCCESS_FILE, handler, null);
+
+        File logDir = new File(LOG_FILE_LOCATION);
+        String[] logFiles = logDir.list();
+        if (ArrayUtils.isNotEmpty(logFiles)) {
+            for (String fileName : logFiles) {
+                if (LOG_FILE_NAME.equals(fileName)) {
+                    isFilePresent = true;
+                }
+            }
+        }
+        assertTrue(isFilePresent);
+    }
+
+    @Test(description = "Method to test the bulk user import error scenario. The test checks for the RemoteException " +
+            "which is thrown by the bulkImportUser method", expectedExceptions = {AxisFault.class},
+            dependsOnMethods = {"testBulkUserImportLogFileIsPresent"})
+    public void testBulkImportWithUserAdminException() throws RemoteException, UserAdminUserAdminException {
+
+        File bulkUserFile = new File(BULK_USER_FILE + BULK_USER_IMPORT_ERROR_FILE);
+
+        DataHandler handler = new DataHandler(new FileDataSource(bulkUserFile));
+        userMgtClient.bulkImportUsers(USER_STORE_DOMAIN, BULK_USER_IMPORT_ERROR_FILE, handler, null);
+
+        File logDir = new File(LOG_FILE_LOCATION);
+        String[] logFiles = logDir.list();
+        if (ArrayUtils.isNotEmpty(logFiles)) {
+            for (String fileName : logFiles) {
+                if (LOG_FILE_NAME.equals(fileName)) {
+                    isFilePresent = true;
```

Comment:
shouldn't we assert something here?

Reply:
Here we check the exception. 
`expectedExceptions = {AxisFault.class}`